.github/workflows/main.yml

```diff
@@ -57,6 +57,26 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
 
+      - name: Push beta app images
+        if: ${{github.ref == 'refs/heads/develop'}}
+        run: |
+          echo ${{ secrets.PAT }} | docker login ghcr.io -u pascal-airy --password-stdin
```

Comment:
we have to use a machine user here so that the credentials don't depend on any team member